app/containers/ActionsList/ActionItem.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { withNavigation } from 'react-navigation';
+import PropTypes from 'prop-types';
+
+import DisclosureIndicator from '../DisclosureIndicator';
+import Touch from '../../utils/touch';
+import { CustomIcon } from '../../lib/Icons';
+
+import { COLOR_WHITE } from '../../constants/colors';
+import styles from './styles';
+
+const ActionItem = ({
+	icon, name, description, type, event, route, params, testID, disabled, navigation
+}) => {
+	const onPressTouchable = () => {

```

Comment:
Is better if you remove `event` and `route` and only pass `onPress` prop, because it can be weird if we send `route + event` props.. IMO.